spec/manager_spec.rb

```diff
@@ -1,9 +1,152 @@
 require "webmock/fixtures"
 
 describe WebMock::Fixtures::Manager do
-  describe "constructor" do
-    it "should not fail" do
-      WebMock::Fixtures::Manager.new()
+  after(:example) do
+    # Ensure we remove any added fixtures between tests
+    WebMock::Fixtures::Manager.reset!
```

Comment:
We should definitely have a test for `reset!` first if it's critical behavior =/